src/xpk/core/docker_manager.py

```diff
@@ -163,11 +163,11 @@ def run_command(
       container = self.client.containers.run(
           image=self.img_name,
           entrypoint=cmd,
-          remove=self.rm_container_after,
-          name=self.container_name,
-          stdout=True,
-          stderr=True,
-          stream=True,
+          remove=True,
```

Comment:
I think passing this as argument was a better way. We can use it for testing etc. Now it is hardcoded

Reply:
We remove it in finally block anyways. better to let the automatic stay there too and try manual delete if it wasn't removed.
What testing scenario may we need to keep the container? Manual test or automated?

Reply:
I thought about not having remove=True, but having remove=remove_container_after(it may be set to true by default).  It may be useful in automated tests.